### DISCUSSION TITLE
打包 power-profiles-daemon

### DISCUSSION THREAD

**Discussion by hualet** [linuxdeepin/developer-center#10858] [Developer Discussion | 开发者讨论]
https://gitlab.freedesktop.org/upower/power-profiles-daemon

一个关于系统电源管理的库，目前看在Ubuntu上是替代 tlp 的，我们需要研究一下效果如何。

**Comment by hualet** [comment]
@Zeno-sole

**Comment by UTsweetyfish** [comment]
关联
https://github.com/deepin-community/power-profiles-daemon/pull/1
https://github.com/deepin-community/Repository-Integration/pull/2200
https://github.com/linuxdeepin/developer-center/issues/10871

**Comment by Zeno-sole** [comment]
已完成
